src/task_monitor/tasks/finalize_identities.rs

```diff
@@ -10,16 +10,14 @@ use crate::database::Database;
 use crate::identity_tree::{Canonical, TreeVersion, TreeWithNextVersion};
 use crate::utils::async_queue::{AsyncPopGuard, AsyncQueue};
 
-// It can take up to 40 minutes to bridge the root
-const FINALIZE_ROOT_SLEEP_TIME: Duration = Duration::from_secs(30);
-
-const MAX_FINALIZATION_ATTEMPTS: usize = 100;
-
 pub struct FinalizeRoots {
     database:          Arc<Database>,
     identity_manager:  SharedIdentityManager,
     finalized_tree:    TreeVersion<Canonical>,
     mined_roots_queue: AsyncQueue<U256>,
+
+    finalization_max_attempts: usize,
+    finalization_attempt_time: Duration,
```

Comment:
Nit: Consider changing this to `finalization_sleep_time` or similar for clarity.

Reply:
Done